### PR TITLE
New function

### PR DESCRIPTION
**Pull request by Mrjulss**
Hab noch eine Funktion hinzugefügt die mehrere lokale Minima hat. Das wurd von den Dozenten gewünscht. Damit lässt sich zeigen, dass Momentum in der Lage ist diese zu überspringen, während Regular Gradient Descent "hängen bleibt".

### CODE REVIEW

**Review by lars-heimann — APPROVED** [review]
sieht gut aus. Bei AdaGrad und RMSProp hab ich das Gefühl dass learning rate scheduling wichtig wäre. Allerdings geht das nur sinnvoll, wenn man s für jeden Scheduler einstellen kann. Ich kann mir das die tage auch mal angucken
